<launch>

  <node type="point_cloud2_assembler" pkg="laser_assembler" name="my_assembler">
    <!-- cloud = topic to recieve clouds on -->
    <remap from="cloud" to="pcl_stitcher_input_processed"/> 
    <param name="max_clouds" type="int" value="400" />
    <param name="tf_cache_time_secs" type="double" value="45.0" />
    <!-- I think this sets the fixed frame to be base_link, and tf takes care of everything else-->
    <param name="fixed_frame" type="string" value="base_link" />
  </node>

  <node name="realsense_processing" pkg="ur_grasping" type="realsense.py">
    <param name="pcl_fwd_status" type="boolean" value="false" />
  </node>

  <include file="$(find ur_grasping)/launch/pcl_filtering.launch"/>

</launch>